libbindgen/src/ir/context.rs

```diff
@@ -959,20 +959,47 @@ impl<'ctx> BindgenContext<'ctx> {
             return *id;
         }
 
-        let module_id = self.next_item_id();
-        let module_name = self.translation_unit
-            .tokens(&cursor)
-            .and_then(|tokens| {
-                if tokens.len() <= 1 {
-                    None
-                } else {
-                    match &*tokens[1].spelling {
-                        "{" => None,
-                        s => Some(s.to_owned()),
+        let mut module_name = None;
+        if let Some(tokens) = self.translation_unit.tokens(&cursor) {
```

Comment:
It would be absolutely lovely if this tokens processing wasn't inline, and instead was a separate helper function.

```rust
fn is_inline_namespace(unit: &clang::TranslationUnit, cursor: &clang::Cursor) -> bool {
    ...
}
```